modules/install/pages/migration.adoc

```diff
@@ -1,80 +1,93 @@
 
 [abstract]
 {description}
-This new feature is supported for *new* buckets that are created in Couchbase Server version 4.6.0.
-You cannot change the conflict resolution mode to the _Timestamp-based Conflict Resolution_ for existing buckets after upgrading to version 4.6.0.
+Couchbase Server supports this feature for new buckets.
+
+To enable timestamp-based conflict resolution for existing data, migrate the data to a new bucket with the timestamp-based conflict resolution type,
+using the `cbbackupmgr` tool.
+For more information about the `cbbackupmgr` tool, see xref:backup-restore:backup-restore.adoc[Backup and Restore].
 
-If you wish to enable the timestamp-based conflict resolution for your existing data, then you must migrate your data to version 4.6.0 cluster using the `cbbackupmgr` tool.
-To learn more about the tool, See xref:backup-restore:backup-restore.adoc[Backup and Restore].
+IMPORTANT: This is a one time migration.
+The bucket without timestamp-based conflict resolution must be removed and a new bucket with the xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc#timestamp-based-conflict-resolution[timestamp-based conflict resolution] type must be created as part of the migration.
 
-IMPORTANT: This is a one time migration and the bucket must be switched to a new conflict resolution type as part of the migration.
+These are 2 scenarios:
 
-To understand more about timestamp-based Conflict Resolution, see xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc#timestamp-based-conflict-resolution[Timestamp-Based Conflict Resolution].
+* Scenario 1: Unidirectional replication for disaster recovery -
+Cluster 1 (Bucket A) continuously replicates data into Cluster 2 (Bucket A'), which is one way.
 
-Here are two scenarios:
+* Scenario 2: Bidirectional replication for data locality -
+Cluster 1 (Bucket A) and Cluster 2 (Bucket A’) replicate data into each other.
 
-* Scenario 1 - In case of unidirectional replication, for example, for a disaster recovery where Cluster 1 (Bucket A) has one way replication streams to Cluster 2 (Bucket A’).
-* Scenario 2 - In case of bidirectional replication, for example, for data locality where Cluster 1 (Bucket A) has both ways replication streams to Cluster 2 (Bucket A’).
+== Migration during Unidirectional Replication
 
-For Scenario 1, perform the following steps to migrate your data to a new cluster:
+Follow these steps to migrate your data to a new bucket during unidirectional replication:
 
 . Stop application traffic coming into Cluster 1 (Bucket A).
 +
-NOTE: Allow __enough tim__e for the replication queues to drain.
-Check the *outbound XDCR mutation* statistics to confirm.
-For instructions, see xref:manage:monitor/ui-monitoring-statistics.adoc#outgoing_xdcr_stats[Monitoring Outgoing XDCR].
+NOTE: Allow enough time for the replication queues to drain.
+Confirm by reviewing the XDCR Total Outbound Mutations statistic.
```

Comment:
Confirm by reviewing the XDCR Total Outbound Mutations statistic, which is derived from xdcr_changes_left_total.